streams/src/test/java/org/apache/kafka/streams/state/internals/GlobalStateStoreProviderTest.java

```diff
@@ -110,6 +111,9 @@ public void before() {
             );
         expect(mockContext.taskId()).andStubReturn(new TaskId(0, 0));
         expect(mockContext.recordCollector()).andStubReturn(null);
+        final Map<String, Object> configValues = new HashMap<>();
+        configValues.put(InternalConfig.IQ_CONSISTENCY_OFFSET_VECTOR_ENABLED, false);
```

Comment:
I think these shouldn't be necessary anymore, since the default is `false`. Can we remove them in a follow-on PR?